lxd/auth/authorization_types.go

```diff
@@ -103,7 +103,7 @@ const (
 	// EntitlementCanViewResources is the `can_view_resources` Entitlement. It applies to entity.TypeServer.
 	EntitlementCanViewResources Entitlement = "can_view_resources"
 
-	// EntitlementCanViewMetrics is the `can_view_metrics` Entitlement. It applies to entity.TypeServer.
+	// EntitlementCanViewMetrics is the `can_view_metrics` Entitlement. It applies to entity.TypeServer and entity.TypeProject.
```

Comment:
Please can you expand the commit message here to explain what this changes/improves/fixes?

Reply:
Fixed, the commit message itself was wrong as it was saying `can_view_metrics` will be extended for both projects and instances.